recipe/meta.yaml

```diff
@@ -35,7 +37,7 @@ about:
   license: BSD
   summary: Delete Anaconda configuration files
   description: |
-    anaconda-clean remove configuration files and directories from Anaconda and its programs.
+    anaconda-clean removes configuration files and directories from Anaconda and its programs.
   doc_url: https://docs.continuum.io/anaconda/install
```

Comment:
Should the doc url point to the readme instead? Perhaps to https://github.com/ContinuumIO/anaconda-clean